deploy/clowdapp.yaml

```diff
@@ -105,7 +105,7 @@ parameters:
   value: "info"
 - name: MIN_REPLICAS
   description: The number of replicas to use in the deployment
-  value: "1"
+  value: "3"
```

Comment:
better to create a MR for stage. 
/lgtm can auto approve it.

Reply:
@bzwei I thought we weren't making MRs for stage anymore?